recurser/string-is
# Add support for SQL formatting

## How to test the PR

1. Load http://localhost:3000/sql-formatter and try some combinations
2. Paste a query into http://localhost:3000/ and check that it is auto-detected


## Checklist

- [x] New functionality has tests.
- [x] README has been updated (if necessary).
- [x] New environment variables have been added to `.env.example` (if necessary).
